.golangci.yml

```diff
@@ -0,0 +1,8 @@
+---
+linters:
+  enable:
+    - goimports
+    - misspell
+  disable:
```

Comment:
Should we disable them in the individual files? I understand that we need `noPositional` in those rules, but we want those checks in other places.

Reply:
Fair point. Done.